requirements/constraints.txt

```diff
@@ -12,4 +12,6 @@
 django<2.3
 
 # jsonfield2>3.0.3 has dropped support for python 3.5
-jsonfield2==3.0.3
\ No newline at end of file
+jsonfield2==3.0.3
+
+celery<4.3
```

Comment:
Should probably add a comment explaining the constraint.  Is there a reason to keep it constrained beyond just making sure tests pass with the next version we're upgrading to?